enterprise/app/controllers/api/v1/accounts/captain/assistants_controller.rb

```diff
@@ -32,6 +32,10 @@ def playground
     render json: response
   end
 
+  def tools
+    @tools = Captain::Assistant.available_agent_tools
```

Comment:
@scmmishra Getting this error, might be `Captain::AssistantPolicy` was missing the `tools?` method.
<img width="2290" height="486" alt="CleanShot 2025-07-14 at 20 08 29@2x" src="https://github.com/user-attachments/assets/0bc67032-5b20-408c-8573-8923d8d05fcc" />